k8s/namespaces/ccd/ccd-regr/ccd-regr.yaml

```diff
@@ -0,0 +1,106 @@
+      idamWebUrl: https://idam-web-public.demo.platform.hmcts.net
+      ccdAdminWebIngress: ccd-admin-{{ .Release.Name }}.demo.platform.hmcts.net
+      ccdApiGatewayIngress: gateway-{{ .Release.Name }}.demo.platform.hmcts.net
+      ccdCaseManagementWebIngress: www-{{ .Release.Name }}.demo.platform.hmcts.net
+      devMode: true
+      environment: demo
+      enableKeyVaults: false
+    ccd-admin-web:
+      nodejs:
+        image: hmctspublic.azurecr.io/ccd/admin-web:prod-13b39f2-20220228181417 #{"$imagepolicy": "flux-system:ccd-admin-web"}
+        ingressClass: traefik-no-proxy
+        ingressHost: ccd-admin-{{ .Release.Name }}.demo.platform.hmcts.net
+        environment:
+          ADMINWEB_IMPORT_AUDITS_URL: http://ccd-regr-ccd-definition-store/health # odd hack to get import working?!?
+        secrets:
+          IDAM_OAUTH2_AW_CLIENT_SECRET:
+            disabled: false
+    ccd-api-gateway-web:
+      nodejs:
+        image: hmctspublic.azurecr.io/ccd/api-gateway-web:prod-374fc8b-20220228181429 #{"$imagepolicy": "flux-system:ccd-api-gateway-web"}
+        ingressClass: traefik-no-proxy
+        ingressHost: gateway-{{ .Release.Name }}.demo.platform.hmcts.net
+        environment:
+          PROXY_CASE_ACTIVITY: "http://{{ .Release.Name }}-ccd-case-activity-api"
+        secrets:
+          IDAM_OAUTH2_CLIENT_SECRET:
+            disabled: false
+    ccd-case-activity-api:
+      nodejs:
+        image: hmctspublic.azurecr.io/ccd/case-activity-api:prod-7092190-20220228181444 #{"$imagepolicy": "flux-system:ccd-case-activity-api"}
+        ingressHost: ccd-case-activity-api-{{ .Release.Name }}.demo.platform.hmcts.net
+        environment:
+          REDIS_HOST: "{{ .Release.Name }}-redis-master"
+          REDIS_PORT: 6379
+          REDIS_PASSWORD: fake-password
+          REDIS_SSL_ENABLED: ""
+        keyVaults:
+      redis:
+        enabled: true
+    ccd-case-print-service:
+      nodejs:
+        image: hmctspublic.azurecr.io/ccd/case-print-service:prod-3219366-20220225104111 #{"$imagepolicy": "flux-system:ccd-case-print-service"}
+    ccd-definition-store-api:
+      java:
+        image: hmctspublic.azurecr.io/ccd/definition-store-api:prod-b76c0a1-20220301174256 #{"$imagepolicy": "flux-system:ccd-definition-store-api"}
+        autoscaling:
+          enabled: false
+        secrets:
+          STORAGE_ACCOUNT_NAME:
+            disabled: false
+          STORAGE_ACCOUNT_KEY:
+            disabled: false
+        environment:
+          AZURE_STORAGE_DEFINITION_UPLOAD_ENABLED: true
+    ccd-data-store-api:
+      java:
+        image: hmctspublic.azurecr.io/ccd/data-store-api:prod-0b87f89-20220301173815 #{"$imagepolicy": "flux-system:ccd-data-store-api"}
+        environment:
+          CCD_DM_DOMAIN: https?://.*dm-store.*
+          ELASTIC_SEARCH_ENABLED: true
+          ELASTIC_SEARCH_HOSTS: "{{ .Release.Name }}-es-master:9200"
+          ELASTIC_SEARCH_DATA_NODES_HOSTS: "http://{{ .Release.Name }}-es-master:9200"
```

Comment:
Are these the only env vars that need to be specified here? At least a few more looks to be required to be overridden, as they are in data store demo.